distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -623,7 +623,7 @@ protected AdminConfig getAdminConfig() {
         return adminConfig;
     }
 
-    protected ServiceManager getServiceManager() {
+    public ServiceManager getServiceManager() {
```

Comment:
❗️We don't need to make this `public` if we pass a reference to the `ServiceManager` instead of a reference to the test class itself (see my comment on the `OpenSearchSourceProperties` class below).